src/pages/index.css

```diff
@@ -26,4 +27,13 @@
 .index__newsletter {
   display: flex;
   justify-content: center;
-}
\ No newline at end of file
+}
+
+@media (max-width: 650px) {
+  .index__headline__delimiter {
```

Comment:
Hehe, this is a convention from work, sorry for not explaining earlier. It's a BEM dialect. 
`.${component}__${partOfComponent}`
In this case it would be `.index__headlineDelimiter`